src/hidpp20.c

```diff
@@ -2412,92 +2421,67 @@ hidpp20_onboard_profiles_parse(struct hidpp20_device *device,
 	crc_valid = hidpp20_onboard_profiles_is_sector_valid(device,
```

Comment:
1/3: If the crc is not valid (corrupted dictionary)..

Reply:
Why I don't understand? What do you want me to do?

Reply:
The three comments 1/3 2/3 and 3/3 are supposed to be considered as a whole :)

Reply:
So the new changes I pushed are fine?